2023-06-01T02:08:15.9641853Z ##[group]Run actions/setup-python@v4
2023-06-01T02:08:15.9642186Z with:
2023-06-01T02:08:15.9642473Z   python-version: 3.8
2023-06-01T02:08:15.9642764Z   architecture: x64
2023-06-01T02:08:15.9642990Z   cache: pip
2023-06-01T02:08:15.9643303Z   check-latest: false
2023-06-01T02:08:15.9643721Z   token: ***
2023-06-01T02:08:15.9644009Z   update-environment: true
2023-06-01T02:08:15.9644277Z   allow-prereleases: false
2023-06-01T02:08:15.9644597Z ##[endgroup]
2023-06-01T02:08:16.1259527Z ##[group]Installed versions
2023-06-01T02:08:16.1392587Z Successfully set up CPython (3.8.16)
2023-06-01T02:08:16.1396338Z ##[endgroup]
2023-06-01T02:08:16.7975060Z [command]/opt/hostedtoolcache/Python/3.8.16/x64/bin/pip cache dir
2023-06-01T02:08:17.1853149Z /home/runner/.cache/pip
2023-06-01T02:08:18.0049863Z Received 6307532 of 6307532 (100.0%), 12.6 MBs/sec
2023-06-01T02:08:18.0050867Z Cache Size: ~6 MB (6307532 B)
2023-06-01T02:08:18.0079890Z [command]/usr/bin/tar -xf /home/runner/work/_temp/2d2fe76d-784d-4517-8d26-0749f8b4c526/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-06-01T02:08:18.0276930Z Cache restored successfully
2023-06-01T02:08:18.0293902Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.16-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
